### DISCUSSION TITLE
How to update from version 23.2 to 23.3

### DISCUSSION THREAD

**Discussion by StoniHome** [john30/ebusd#1151] [Q&A]
Hi

I've connected my ebusd v5 to an raspi over GPIO.

What is the easiest way to update from version 23.2 to version 23.3

**Comment by StoniHome** [comment]
I figured it out by myself. Now I am on 23.3

**Reply by boskape** [reply]
for the HA add on ?

**Comment by Trapper12** [comment]
Hi StoniHome,

and how did you do it?

was it:
sudo apt-get update
sudo apt-get install ebusd
Regards
Klaus

**Comment by StoniHome** [comment]
Hi Klaus

Simply do

Sudo apt-get update
Sudo apt-get upgrade

Cu

**Reply by Trapper12** [reply]
:-) Thanks

**Reply by Trapper12** [reply]
Sudo apt-get upgrade is enough?

**Reply by boskape** [reply]
also for the HA Addon

**Reply by githuber110** [reply]
Can you give me some more details how to get their where I need to run these functions?

**Comment by nobbilie** [comment]
How does this work under Home Assistant?
I always get a message in the log that there is a newer version.
